cirq/contrib/quimb/mps_simulator.py

```diff
@@ -29,35 +29,60 @@
 from cirq.sim import simulator
 
 
+class MPSOptions:
```

Comment:
Consider using `@dataclasses.dataclass` for this.

Reply:
Done.

Reply:
`@dataclasses.dataclass` automatically creates the `__init__` method (and several others) from variable definitions - see the [docs](https://docs.python.org/3/library/dataclasses.html) for details. With that in mind, it should be possible to condense this to:
```
@dataclasses.dataclass
class MPSOptions:
    """Put the __init__ docstring here"""
    # ...Quimb docs link...
    method: str = 'svds',
    max_bond: Optional[int] = None,
    cutoff_mode: str = 'rsum2',
    cutoff: float = 1e-6,
    sum_prob_atol: float = 1e-3,
```

Reply:
also use `frozen=True`

Reply:
Neat. Done and Done.